apps/src/applab/applab.js

```diff
@@ -692,7 +692,7 @@ Applab.init = function (config) {
     isCurriculumLevel: isCurriculumLevel(config.level.validationEnabled),
   });
 
-  config.dropletConfig = dropletConfig;
+  config.dropletConfig = {...dropletConfig};
```

Comment:
With the jest migration, it enabled certain modules to be imported as ESM instead of CommonJS. The Applab init code imports `dropletConfig` which is then passed to `Immutable.fromJS`. This function will return an Immutable for a CommonJS import, but for ESM it will return the same input which causes the subsequent call `mergeDeepWith` to fail (see below stack).

To workaround this while being compatible with CommonJS and ESM, I do a shallow clone by spreading the object which converts it from an ES Module to a plain old JS object as [recommended here](https://stackoverflow.com/a/48510588).

 Without this change, the unit test fails with:

```
  ● Applab › Applab.init() › configuration options › hasDataMode page constant › is false in widget mode

    TypeError: baseImmutable.mergeWith is not a function

      538 |
      539 |   var baseImmutable = Immutable.fromJS(baseObject);
    > 540 |   return baseImmutable.mergeWith(deepConcatMerger, overrides).toJS();
          |                        ^
      541 | }
      542 |
      543 | /**

      at Object.mergeWith [as deepMergeConcatArrays] (src/utils.js:540:24)
      at Object.deepMergeConcatArrays [as init] (src/applab/applab.js:738:34)
      at Object.init (test/unit/applab/applabTest.js:691:18)
```